Dockerfile.prod

```diff
@@ -0,0 +1,21 @@
+FROM node:12
+
+RUN mkdir -p /app
+WORKDIR /app
+
+# copy the project code
+COPY src /app/src
+COPY static /app/static
+COPY .env .eslintrc package.json package-lock.json rollup.config.js /app/
```

Comment:
I don't see the `.env` file in Git, is that supposed to be this way?

Reply:
Don't really see a point to list all the files. Copying `COPY .` could be enough

Reply:
It would copy everything, while the intention was only to copy certain files

Reply:
Copying `.env` was the only way to fix the "process is undefined" error in production mode. I couldn't understand why, but it did. Also, `.env` files are not supposed to be checked into version control, but they are required to run the project properly. I was meaning to add this to the README (of the main repo) but forgot.
@VanishMax as Lev pointed out, that would also copy node_modules, .gitignore, Dockerfile, and many more files we don't need